x-pack/test/security_solution_cypress/cypress/tasks/api_calls/cases.ts

```diff
@@ -28,3 +30,23 @@ export const createCase = (newCase: TestCase) =>
     },
     headers: { 'kbn-xsrf': 'cypress-creds', 'x-elastic-internal-origin': 'security-solution' },
   });
+
+export const getCases = () =>
```

Comment:
It would make it easier later if we are explicit with the signature types
```suggestion
export const getCases = async (): Promise<CasesFindResponse> =>
```